configs/IFLIGHT_2RAW_H743/config.h

```diff
@@ -0,0 +1,140 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU       STM32H743
+
+#define BOARD_NAME          IFLIGHT_2RAW_H743
+#define MANUFACTURER_ID     IFRC
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_MAX7456
+#define USE_SDCARD
+
+#define BEEPER_PIN              PC9
+#define MOTOR1_PIN              PB0
+#define MOTOR2_PIN              PB1
+#define MOTOR3_PIN              PA0
+#define MOTOR4_PIN              PA1
+#define MOTOR5_PIN              PA2
+#define MOTOR6_PIN              PA3
+#define MOTOR7_PIN              PD12
+#define MOTOR8_PIN              PD13
+#define LED_STRIP_PIN           PA8
+#define SERVO1_PIN              PD14 //  S9
+#define SERVO2_PIN              PD15 // S10
+#define SERVO3_PIN              PE5  // S11
+#define SERVO4_PIN              PE6  // S12
+#define UART1_TX_PIN            PA9
+#define UART2_TX_PIN            PD5
+#define UART3_TX_PIN            PD8
+#define UART4_TX_PIN            PB9
+#define UART6_TX_PIN            PC6
+#define UART7_TX_PIN            PE8
+#define UART8_TX_PIN            PE1
+#define UART1_RX_PIN            PA10
+#define UART2_RX_PIN            PD6
+#define UART3_RX_PIN            PD9
+#define UART4_RX_PIN            PB8
+#define UART6_RX_PIN            PC7
+#define UART7_RX_PIN            PE7
+#define UART8_RX_PIN            PE0
+#define I2C1_SCL_PIN            PB6
+#define I2C1_SDA_PIN            PB7
+#define I2C2_SCL_PIN            PB10
+#define I2C2_SDA_PIN            PB11
+#define LED1_PIN                PE3
+#define LED2_PIN                PE4
+#define SPI1_SCK_PIN            PA5
+#define SPI2_SCK_PIN            PB13
+#define SPI3_SCK_PIN            PB3
```

Comment:
duplicate

Reply:
Thank you for the comments. I am aware of these duplicate defines, but this is what is shown on the most recent schematic. :) I have marked this as a draft until we get more details from iFlight.

Reply:
Okay 😁

Reply:
`PC10`, `PC11` and `PC12` seem to be the correct pins according to the new schematic